### ISSUE TITLE
No such driver version 115.0.5790.110 for linux64

### ISSUE THREAD

**Issue by Priyathangaraj95** [SergeyPirogov/webdriver_manager#576]
The corresponding chrome driver is not getting installed for this chrome version 115.0.5790.110 (Debian linux)

Error logs:
 
  File "/root/workspace/gherkin-web/sqa_web/driverconfig.py", line 65, in build_chrome
    executable = ChromeDriverManager().install()
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/.pyenv/versions/3.11.0/lib/python3.11/site-packages/webdriver_manager/chrome.py", line 39, in install
    driver_path = self._get_driver_binary_path(self.driver)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/.pyenv/versions/3.11.0/lib/python3.11/site-packages/webdriver_manager/core/manager.py", line 33, in _get_driver_binary_path
    file = self._download_manager.download_file(driver.get_driver_download_url())
                                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/.pyenv/versions/3.11.0/lib/python3.11/site-packages/webdriver_manager/drivers/chrome.py", line 59, in get_driver_download_url
    modern_version_url = self.get_url_for_version_and_platform(driver_version_to_download, os_type)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/.pyenv/versions/3.11.0/lib/python3.11/site-packages/webdriver_manager/drivers/chrome.py", line 94, in get_url_for_version_and_platform
    raise Exception(f"No such driver version {browser_version} for {platform}")

**Comment by yadalik** [comment]
Are you using latest version of  webdriver_manager?

**Comment by eternalharvest** [comment]
same issue!

it seems that `chromedriver` corresponding to the latest Google Chrome 115 is not yet released.
when I fetch latest release from official chromedriver repository, it says `114.0.5735.90` is the latest version.

https://chromedriver.storage.googleapis.com/LATEST_RELEASE

**Comment by TomaszKarski** [comment]
same issue for me on Windows and Mac

**Comment by Priyathangaraj95** [comment]
> Are you using latest version of webdriver_manager?
Yes - 3.9.1

**Comment by Dipnoid** [comment]
@eternalharvest Chrome driver changed their json api endpoints starting with version 115, checkout [here](https://github.com/GoogleChromeLabs/chrome-for-testing#json-api-endpoints)

**Comment by yadalik** [comment]
> > Are you using latest version of webdriver_manager?
> > Yes - 4.0.0

Hm, strange, for me it worked again for chrome 115 after `pip install --upgrade webdriver_manager` to 4.0.0 (I noticed in e-mail you said 3.9.1 installed), and I assumed it was fixed in https://github.com/SergeyPirogov/webdriver_manager/issues/566 
Still not working for you?

**Comment by LkSandeep** [comment]
Chrome Version 115.0.5790.110 (Official Build) (64-bit)
Even after updating to latest webdriver-manager 4.0.0
`"webdriver-manager": {
            "hashes": [
                "sha256:05f58ab599b907b03e0aa8d6b43624b1ca8c12d5d48455c79c531b9052ae2356",
                "sha256:2b8d1c2cb590d296ade18bc7bc2589578a868f5587133922d744641f6a757a20"
            ],
            "index": "pypi",
            "version": "==4.0.0"
        }`
        
I am getting below error

`        except OSError as err:
            if err.errno == errno.ENOENT:
>               raise WebDriverException(
                    "'%s' executable needs to be in PATH. %s" % (
                        os.path.basename(self.path), self.start_error_message)
E                       selenium.common.exceptions.WebDriverException: Message: 'chromedriver' executable needs to be in PATH. Please see https://sites.google.com/a/chromium.org/chromedriver/home
`

**Comment by Priyathangaraj95** [comment]
> > > Are you using latest version of webdriver_manager?
> > > Yes - 4.0.0
> 
> Hm, strange, for me it worked again for chrome 115 after `pip install --upgrade webdriver_manager` to 4.0.0 (I noticed in e-mail you said 3.9.1 installed), and I assumed it was fixed in #566 Still not working for you?

It's working now for me.. Thanks

**Comment by kyoliu** [comment]
i also have same problem , on my mac my chrome 115.0.5790.114(arm64) 
```
selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fa3d1161850>
response = {'status': 500, 'value': '{"value":{"error":"unknown error","message":"unknown error: cannot find Chrome binary","stac...ff81b91d4e1 _pthread_start + 125\\n19  libsystem_pthread.dylib             0x00007ff81b918f6b thread_start + 15\\n"}}'}
```
it see like can not find driver for chrome
